src/main/java/com/rultor/agents/ecs/StartsECS.java

```diff
@@ -79,14 +82,14 @@ public StartsEC2(final Amazon amaz) {
     @Override
     //@todo #629 Add Instance params to Directive, for example publicIpAddress
     public Iterable<Directive> process(final XML xml) throws IOException {
-        final Instance instance = this.amazon.runOnDemand();
+        final ContainerInstance instance = this.amazon.runOnDemand();
         Logger.info(
             this,
-            "EC2 instance %s created",
+            "ECS instance %s created",
             instance
         );
         return new Directives().xpath("/talk")
-            .add("ec2")
-            .attr("id", instance.getInstanceId());
+            .add("ecs")
+            .attr("id", instance.getEc2InstanceId());
```

Comment:
@original-brownbear But in:

http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/ecs/model/Container.html

there is no method to get instance identifier. Do I have to use `getName` instead?